common/source/docs/common-holybro-ph5x.rst

```diff
@@ -79,11 +79,11 @@ UART Mapping
 ============
 
  - SERIAL0 -> USB 
- - SERIAL1 -> UART7 (Telem1) RTS/CTS pins
- - SERIAL2 -> UART5 (Telem2) RTS/CTS pins
+ - SERIAL1 -> UART7 (Telem1) RTS/CTS pins (DMA enabled)
+ - SERIAL2 -> UART5 (Telem2) RTS/CTS pins (DMA enabled)
  - SERIAL3 -> USART1 (GPS1)
```

Comment:
The GPS ports are not DMA-enabled and this is an F7 - that might be a problem

Reply:
above my paygrade....what should we do, if anything....

Reply:
Think your change is fine for now